cfripper/config/config.py

```diff
@@ -72,13 +72,46 @@ class Config:
         "ec2:CreateDhcpOptions",
         "ec2:CreateCustomerGateway",
         "ecs:*",
+        # Lambda
```

Comment:
Should we include Url permissions such as the ones being updated in https://github.com/Skyscanner/pycfmodel/pull/98 `tests/test_action_expander.py`?

Reply:
Are you sure this actions allow resource level constraints? A lot of services do not allow it for create actions

Reply:
did you figure out if it's needed to add the new cf actions?

Reply:
done